bodhi/server/ffmarkdown.py

```diff
@@ -22,6 +22,10 @@
 Author: Ralph Bean <rbean@redhat.com>
 """
 
+import re
+import typing
+import xml
```

Comment:
We can move the ```re``` and ```xml``` imports to an ```if typing.TYPE_CHECKING``` block, and then reference them using strings below. This way it's clearer that this module doesn't directly use these libraries and that they are only referenced during type checking.